bin/qlf_models.py

```diff
@@ -30,19 +31,18 @@ def insert_exposure(
         """ Inserts and gets exposure and night if necessary. """
 
         # Check if expid is already registered
-        if not Exposure.objects.filter(expid=expid):
+        if not Exposure.objects.filter(exposure_id=expid):
             exposure = Exposure(
-                expid=expid, night=night,
+                exposure_id=expid, night=night,
```

Comment:
@crisingulani do `expid` and `exposure_id` have the same meaning here? if so would suggest to use the same variable name.